LICENSE.build

```diff
@@ -1,19 +1,22 @@
-Copyright (c) 2020 The Meson development team
```

Comment:
Should we update the template as well?
https://github.com/mesonbuild/mesonwrap/blob/master/mesonwrap/tools/repoinit.py#L36-L55

Reply:
I copied upstream's `LICENSE` file for this one as I did it by hand.